addons/sale_timesheet/models/account.py

```diff
@@ -147,7 +147,9 @@ def _timesheet_get_sale_domain(self, order_lines_ids, invoice_ids):
             ('timesheet_invoice_type', 'in', ['billable_time', 'non_billable']),
             '&',
             ('timesheet_invoice_type', '=', 'billable_fixed'),
-            ('so_line', 'in', order_lines_ids.ids)
+                '&',
+                ('so_line', 'in', order_lines_ids.ids),
+                ('timesheet_invoice_id', '=', False)
```

Comment:
Cosmetic changes
```suggestion
                ('timesheet_invoice_id', '=', False),
```

Also the commit message should only mention sale_timesheet since helpdesk is a enterprise feature.